shared/login/register/paper-key/index.js

```diff
@@ -34,5 +34,5 @@ PaperKey.propTypes = {
 }
 
 export default connect(
-  (state, ownProps) => ownProps.mapStateToProps(state)
+  state => ({waitingForResponse: state.login.waitingForResponse})
```

Comment:
injecting it this way is fine, it just needs to be passed into the render component